controller.xql

```diff
@@ -91,6 +92,16 @@ else if (ends-with($exist:resource, ".html")) then
         </view>
     </dispatch>
 
+else if (contains($exist:path, "/public/") and ends-with($exist:resource, ".xar")) then
+    <dispatch xmlns="http://exist.sourceforge.net/NS/exist">
+        <forward url="../../public-repo-data/packages/{$exist:resource}"/>
+    </dispatch>
+
+else if (contains($exist:path, "/public/") and (ends-with($exist:resource, ".png") or ends-with($exist:resource, ".svg"))) then
```

Comment:
TODO check if we support other icon image formats

Reply:
No we do not. Well at least it would have been only by accident if, not officially.

Reply:
`icon.svg` is supported not sure why you think it isn’t

Reply:
@duncdrum The code here assumes png and svg. The only question is whether we need to serve others like jpg and gif.

Reply:
Ah I misinterpreted your comment, svg and png are the only supported formats.

Reply:
Got it. Thanks!